csi/deployment.go

```diff
@@ -117,6 +117,8 @@ func NewProvisionerDeployment(namespace, serviceAccount, provisionerImage, rootD
 			"--leader-election",
 			"--leader-election-namespace=$(POD_NAMESPACE)",
 			"--default-fstype=ext4",
+			"--enable-capacity",
+			"--capacity-ownerref-level=2",
```

Comment:
Could you elaborate on why `capacity-ownerref-level` is set to 2?

Reply:
Figured out the answer.

When the external-provisioner creates a `CSIStorageCapacity` object with `--capacity-ownerref-level=2`, it will add an OwnerReference pointing to its Deployment. This ensures that if the external-provisioner's Deployment is deleted, the associated capacity information will also be automatically cleaned up, preventing resource leakage.

Reply:
Figured out the answer.

When the external-provisioner creates a `CSIStorageCapacity` object with `--capacity-ownerref-level=2`, it will add an OwnerReference pointing to its Deployment. This ensures that if the external-provisioner's Deployment is deleted, the associated capacity information will also be automatically cleaned up, preventing resource leakage.

Correct?

Reply:
yes, right.